tests/system/src/test/java/test/javafx/scene/text/TestHandlingBigGlyphID_Win10.java

```diff
@@ -0,0 +1,38 @@
+import javafx.application.Application;
+import static javafx.application.Application.launch;
+import javafx.scene.Group;
+import javafx.scene.Scene;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Font;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextFlow;
+import javafx.stage.Stage;
+
+public class TestHandlingBigGlyphID_Win10 extends Application {
+    @Override
+    public void start(Stage stage) throws Exception {
+        final String fontName = "ARIALUNI.TTF"; // Arial Unicode MS
+        // download from https://www.wfonts.com/download/data/2015/05/16/arial-unicode-ms/ARIALUNI.TTF
+        //  and place in $(user.home)/fonts/
```

Comment:
An automated test should not depend on an external font that needs to be downloaded.

Reply:
This bug occurs only when all of the following conditions are satisfied.
1. on Windows
2. font has glyphID bigger than 32767 (max value of signed short)
3. font has complex character. (isComplexCharCode() return true)

When display characters of above conditions
, Java processes layout() in DWGlyphLayout.java
, then iglyphs[] have minus value by missing MASK.

Windows default does not have such font which is satisfied above conditions.
As far as I know, such font is "Arial Unicode MS" supplied by Microsoft Office.